packages/src/lib/navigation/navigation.component.scss

```
@@ -3,55 +3,162 @@
 @use '../../../core/src/theme/colors';
 @use '../../../core/src/theme/palettes';
 
-.navigation-header {
-  background-color: colors.$blue-dark;
-}
+$desktopRowHeight: 72px;
+$mobileRowHeight: 40px;
 
-.row {
-  min-height: 72px;
+.sdg-navigation {
+  &-header {
+    color: white;
+    background-color: colors.$blue-medium;
+    width: 100%;
 
-  @include layout.breakpoint-down(mobile) {
-    min-height: 40px;
-  }
-}
+    &-container {
+      padding: 0 16px;
+
+      @include layout.breakpoint-down(mobile) {
+        padding: 0 8px;
+      }
+    }
+
+    &-row {
+      display: flex;
+      align-items: center;
+      min-height: $desktopRowHeight;
+      height: $desktopRowHeight;
+
+      &.--two-lines {
+        flex-direction: column-reverse;
+
+        > * {
+          height: $mobileRowHeight;
+        }
+
+        .sdg-navigation-header-actions {
+          align-self: flex-end;
+        }
+      }
+
+      @include layout.breakpoint-down(mobile) {
+        height: fit-content;
+        min-height: $mobileRowHeight;
+      }
+    }
+
+    &-actions {
+      @include layout.breakpoint-down(mobile) {
+        margin-bottom: 0;
+      }
+
+      ::ng-deep {
+        button {
+          margin-left: 24px;
+
+          @include layout.breakpoint-down(mobile) {
+            margin-left: 16px;
+          }
+        }
+
+        .mat-mdc-icon-button {
+          margin-left: 4px;
+        }
+      }
+    }
 
-.nav-link {
-  letter-spacing: 0;
-  font-size: 18px;
-  line-height: 28px;
+    .mdc-tab {
+      --mdc-secondary-navigation-tab-container-height: #{$desktopRowHeight};
 
-  &.active {
-    font-weight: bold;
+      min-width: fit-content;
+      padding-left: 8px;
+      padding-right: 8px;
+
+      @include layout.breakpoint-down(mobile) {
+        --mdc-secondary-navigation-tab-container-height: #{$mobileRowHeight};
+        padding-left: 4px;
+        padding-right: 4px;
+      }
+    }
+
+    ::ng-deep .mat-mdc-tab-links {
+      align-items: center;
+
+      & > * {
```

```suggestion
      & .mdc-tab-indicator {
```
Il ne doit pas y avoir d'espace entre les onglets. Après avoir ajusté le padding de `.mdc-tab`, mettre le padding plutôt sur l'indicateur.